provider/status_change.json

```diff
@@ -0,0 +1,169 @@
+{
+  "definitions": {},
+  "$id": "https://github.com/CityofLosAngeles/mobility-data-specification/blob/master/provider/status_change",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "The MDS Provider Schema, status_change payload",
+  "type": "object",
+  "required": [
+    "version",
+    "data"
+  ],
+  "properties": {
+    "version": {
+      "$id": "#/properties/version",
+      "$ref": "common.json#/definitions/version"
+    },
+    "data": {
+      "$id": "#/properties/data",
+      "type": "object",
+      "title": "The page of data",
+      "required": [],
```

Comment:
Unless I'm missing something, shouldn't `"status_changes"` be inside `"required"` here, since we are saying this is the spec for that payload specifically?